techniques/failures/F97.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Failure of Success Criterion 2.5.6 due to interactions being limited to touch only on touchscreen devices</title>
```

Comment:
The title is a bit confusing. I was rereading it to understand if you meant "due to interactions being limited to touch-only on touchscreen devices" or "due to interactions being limited to touch, only on touchscreen devices".

Reply:
changed to "touch-only" in the title and heading - hopefully this clarifies it (otherwise it may need to be turned around / be made longer to explain fully)